packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -269,6 +270,19 @@ class EditableText extends StatefulWidget {
   /// Defaults to the ambient [Directionality], if any.
   final TextDirection textDirection;
 
+  /// Specifies how platforms may automatically capitialize text entered by the
```

Comment:
I think you should explain more about what "automatically" means in your doc comments (here, and elsewhere): mention that it is dependent upon locale, etc. Does it come from a Dart feature that you can link to, or is it backed by an Android/iOS feature?

Reply:
I updated the documentation to specify that this configures how the platform selects an uppercase vs lowercase keyboard - I think this is more clear